mass: 2.083
rotor_num: 4.0
rotor_c: 8.54858e-06
Kff_v: 1.0
Kff_a: 0.0
tilt_angle: 0.0

pid_x:
  Kp: 2.0 #1.4 
  Ki: 0.0 # 0
  Kd: 0.0 # 0.05
  limit:
    high: 10.0
    low: -10.0
pid_vx:
  Kp: 0.5 #0.3
  Ki: 0.1 # 0.0
  Kd: 0.0 # 0.0
  limit:
    high: 1.57
    low: -1.57

pid_y:
  Kp: 2.0
  Ki: 0.0
  Kd: 0.0
  limit:
    high: 10.0
    low: -10.0
pid_vy:
  Kp: 0.5
  Ki: 0.1
  Kd: 0.0
  limit:
    high: 1.57
    low: -1.57

pid_z:
  Kp: 10 #2.5
  Ki: 0.5 #0.0
  Kd: 0.0 #0.2
  limit:
    high: 10.0
    low: -10.0
pid_vz:
  Kp: 350 #250.0
  Ki: 40 #20.0
  Kd: 0.0
  limit:
    high: 1450.0
    low: -1450.0